examples/async/containers/AsyncApp.js

```diff
@@ -50,7 +50,7 @@ class AsyncApp extends Component {
             </span>
           }
           {!isFetching &&
-            <a href='#'
+            <a href="#"
```

Comment:
It's a bit weird, are we using `"` for JSX strings but `'` for other strings? Why?